tutorials/git/create-branch.md

```diff
@@ -0,0 +1,114 @@
+---
+permalink: /tutorials/git/create-branch.html
+layout: tutorial
+---
+Creating a new branch allows you to add commits on a parallel version of the repository.
+This can be used to test new features and not affect the `main` branch.
+Later, the changes made in the branch can be added to the `main` branch once they are tested.
+
```

Comment:
Why not add a visual example of how branching looks like, meaning a branching graph. A clear one, where a branching is shown. Don't think showing merge is relevant.

Reply:
Okay, you can copy the image from the index.